front/app/containers/SearchPage/components/CrumbsBar.tsx

```diff
@@ -618,94 +621,111 @@ class CrumbsBar extends React.Component<CrumbsBarProps, CrumbsBarState> {
                             }>
                             Bulk Update <FontAwesome name="truck" />
                           </ThemedButton>
-                        ) : null
-                      }
-                    </CurrentUser>
-                  </Form>
-                </Col>
-                <div>
-                  <b>Total Results:</b> {this.props.totalResults} studies
-                </div>
-              </Row>
-              {showCrumbsBar ? (
-                // having trouble getting the theme applied to these ListGroups
-                // <ThemeProvider>
-                //   {theme => (
-                <Row>
-                  <Col
-                    md={12}
-                    style={{
-                      padding: '10px 0px',
-                      display: 'flex',
-                      flexWrap: 'wrap',
-                    }}>
-                    <ListGroup
-                      style={{
-                        display: 'flex',
-                        flexWrap: 'wrap',
-                        border: '1px solid #ddd',
-                        borderRadius: ' 5px',
-                        background: '#fff',
-                        width: '100%',
-                      }}>
-                      <ListGroupItem
-                        style={{
-                          minWidth: '100%',
-                          background: this.props.theme.button,
-                          color: '#fff',
-                        }}
-                        onClick={this.toggleShowFilters}>
-                        {' '}
-                        Filters:{' '}
-                        {Array.from(
-                          this.mkDefaultClearButtons(this.props.searchParams)
-                        )}
-                        {this.state.showFilters ? (
-                          <b>
-                            <FontAwesome
-                              className="chevron-up"
-                              name="chevron-up"
-                              style={{
-                                cursor: 'pointer',
-                                color: '#fff',
-                                margin: '0 0 0 3px',
-                                float: 'right',
-                              }}
-                              // onClick={() => this.toggleShowValue()}
+                        ) : null}
+                      </Form>
+                    </Col>
+                    <Col xs={4} md={4}>
+                      {user && (
```

Comment:
We won't show the export to CSV button without a logged in user. This will help prevent abuse.

Reply:
Not sure we want to, but how hard is it to show it but prompt login to use it.  Reason would be to highlight functionality that is available to logged in users.